### ISSUE TITLE
Strange Safari-only issue

### ISSUE THREAD

**Issue by geoidesic** [oblador/angular-scroll#197]
The examples here work fine in Safari for me: http://oblador.github.io/angular-scroll/

However, when I've implemented the `duScroll` component in my own application, it doesn't work on Safari. Even more strange... if I open the app in both Chrome and Safari at the same time, when I click the scroll trigger in Chrome, then the app scrolls in both Chrome and Safari at the same time. So just to be clear... if I trigger the scroll from the app in Safari, nothing happens but if I trigger it from Chrome then the scroll happens in the Safari window.

How is that even possible?

I've noticed that if I just use the scroll-wheel (or track-pad in this case) to scroll on either Chrome or Safari, then both instances of the app (i.e. in both Safari and Chrome) will scroll simultaneously.

Most confused. Any help appreciated.

**Comment by geoidesic** [comment]
Update: 
I've discovered the cause. In my case this was due to superfluous CSS. So for example:
HTML:
```
<a href="#warning-{{$scope.color}}-0" du-smooth-scroll du-scroll-spy offset="50">
   <div class="animator"> click me </div>
</a>
```
CSS:
```
div.animator{
   transition: all 0.6s cubic-bezier(0.25, 0.46, 0.45, 0.94);
   transition: opacity 0.6s cubic-bezier(0.25, 0.46, 0.45, 0.94);
}
```
While Chrome is ok with this Safari doesn't seem to like it. I don't get any errors reported but it causes duScroll not to work. If I change the CSS to this instead:
div.animator{
   transition: all 0.6s cubic-bezier(0.25, 0.46, 0.45, 0.94);
   // transition: opacity 0.6s cubic-bezier(0.25, 0.46, 0.45, 0.94);
}

Then it works fine.
